### DISCUSSION TITLE
Add Wails to an existing project & Use wails conditionally on the project

### DISCUSSION THREAD

**Discussion by kodeasura** [wailsapp/wails#3156] [Q&A] [answered]
I have a small project made with echo with just some vanilla html with go templates and a sqlite file. I want to setup wails in a way where I will still be able to deploy the normal project to a webserver and conditionally choose to build with wails for the webview desktop apps. Is that something that's possible? 

What am trying to achieve is to have a single code base for both the desktop apps and the website.

**Comment by leaanthony** [comment, marked as answer]
We're looking into a native solution for this but for now you may be able to use the asset server https://wails.io/docs/reference/options/#assetserver
